cmd/deploy/deploy.go

```diff
@@ -179,8 +179,25 @@ func Deploy(ctx context.Context) *cobra.Command {
 			}
 			startTime := time.Now()
 			err = c.RunDeploy(ctx, options)
-			duration := time.Since(startTime)
-			analytics.TrackDeploy(err == nil, utils.IsOktetoRepo(), err, duration, c.PipelineType)
+
+			deployType := "custom"
+			if options.Manifest.IsV2 &&
+				options.Manifest.Deploy != nil &&
+				options.Manifest.Deploy.Compose != nil &&
+				options.Manifest.Deploy.Compose.Manifest != nil {
+				deployType = "compose"
+			}
+			analytics.TrackDeploy(analytics.TrackDeployMetadata{
+				Success:                err == nil,
+				IsOktetoRepo:           utils.IsOktetoRepo(),
+				Duration:               time.Since(startTime),
+				PipelineType:           c.PipelineType,
+				DeployType:             deployType,
+				IsPreview:              false, // TODO: How should we get this value?
```

Comment:
@jLopezbarb is there a way to know this here?

Reply:
The only thing I can think of is checking if its an okteto cluster and then call to the API to check if exists the namespace as preview or namespace

Reply:
Possible options I see are:
* Use client-go library to get the ns information and check annotations/labels (scope). I don't like it because we would be coupling the CLI with the logic of checking the scope to know if it is a preview env or a normal namespace and it would be harder to remove the scope in the future
* Pass an environment variable to the pipeline job specifying if it is a normal pipeline or a preview environment. In that way, that environment variable would be injected in the `okteto deploy` command and it would be available here to check